src/sidebar/views/overview.ts

```diff
@@ -0,0 +1,138 @@
+import { format } from 'node:path';
+import { Command, Event, EventEmitter, ExtensionContext, TreeDataProvider, TreeItem, TreeView, window } from 'vscode';
+import { ExtensionApi } from '../../backend/api';
+import { CheckerMetadata } from '../../backend/types';
+
+export class OverviewItem {
+    constructor(private label: string | (() => string), private command?: Command) {}
+
+    getTreeItem(): TreeItem | Promise<TreeItem> {
+        const label = typeof this.label === 'string' ? this.label : this.label();
+
+        const node = new TreeItem(label);
+        node.command = this.command;
+        node.description = this.command?.tooltip;
+
+        return node;
+    }
+}
+
+export class OverviewView implements TreeDataProvider<string> {
+    private tree?: TreeView<string>;
+
+    // FIXME: Export the keys into an enum, or otherwise check in the itemsList if an entry exists
+    private items: {[id: string]: OverviewItem} = {
+        'loading': new OverviewItem('Loading overview, please wait...'),
+
+        'notfound': new OverviewItem('CodeChecker run not found.'),
+        'notfound2': new OverviewItem('Run CodeChecker, reload metadata,'),
```

Comment:
```suggestion
        'notfound2': new OverviewItem('Run CodeChecker, reload metadata.'),
```

Reply:
This is part of a single block of text, but it was too long to fit on one line.